### ISSUE TITLE
Crashlog after update from 2.92 - 3.11

### ISSUE THREAD

**Issue by pabluuz** [McJty/RFTools#235]
tl;dr:
http://pastebin.com/ixHTm8tW
after updating from 2.92 - 3.11

long story:
I'm running small server for friend. We were using 2.92, until we decided to update rftools to 3.11.
I wasn't able to play, when i logged in, client seemed to not receive map properly from server, and I was disconnected after like 10 seconds.
I've tested it in SSP. I wasn't able to create new world with 3.11 or 3.10. I was able to load world generated without rftools.
For time being, I've downgraded to 3.02, and it seems to work fine, so it's not dead-end for me.
I'll provide You with all my logs, I hope that they'll help You somehow.
http://pabluuz.pl/filesystem/

**Comment by pabluuz** [comment]
Nvm. Sorry, it's not RFtools. Looks like problem is with some other mod.
